opcodes/riscv-opc.c

```diff
@@ -2177,6 +2178,32 @@ const struct riscv_opcode riscv_opcodes[] =
 {"vt.maskc",    0, INSN_CLASS_XVENTANACONDOPS, "d,s,t", MATCH_VT_MASKC, MASK_VT_MASKC, match_opcode, 0 },
 {"vt.maskcn",   0, INSN_CLASS_XVENTANACONDOPS, "d,s,t", MATCH_VT_MASKCN, MASK_VT_MASKCN, match_opcode, 0 },
 
+/* Vendor-specific CORE-V instructions. */
```

Comment:
Move just before `/* Vendor-specific (T-Head) XTheadBa instructions.  */` so that the vendors are in alphabetical order.